app/models/settings/general.rb

```diff
@@ -0,0 +1,136 @@
+# Settings based on RailsSettings models,
+# see <https://github.com/huacnlee/rails-settings-cached> for further info
```

Comment:
Praise: thanks for linking to the inspiration, really helpful for understanding this class!

Reply:
I can't take any credit for this, this was already in `SiteConfig`, I just moved it over :-) I assume the real credit here goes to @rhymes.